pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -119,6 +119,15 @@ def hoc_get_locale_code
   Languages.get_hoc_locale_by_unique_language(@language)
 end
 
+# hourofcode.com calls this to translate tutorial's languages attribute
```

Comment:
This should also mention `code.org` when that's fixed, and we could specifically mention that it's used on `/learn` on each of those sites.

Reply:
Good callout. Fix in [this PR](https://github.com/code-dot-org/code-dot-org/pull/43466).